### ISSUE TITLE
Library view / bookshelf, horizontal scrolling lists of cover images: previous/next arrows, no accessible labels

### ISSUE THREAD

**Issue by danielweck** [readium/readium-desktop#585]

**Comment by Scarsniik** [comment]
It is made on purpose because when you use the keyboard navigation, the scroll is automatique then the arrow is useless. But maybe it should be hidden from screen readers to avoid misunderstanding

**Comment by danielweck** [comment]
https://github.com/readium/readium-desktop/blob/ab47ae557ad9981fea11695fab9b634e10376c80/src/renderer/components/utils/Slider.tsx#L88-L90

https://github.com/readium/readium-desktop/blob/ab47ae557ad9981fea11695fab9b634e10376c80/src/renderer/components/utils/Slider.tsx#L99-L101

Screen readers are not the only type of assistive technology, so to cater for all kinds of interactions I would suggest labelling user interface controls appropriately (semantically and/or with human-readable strings), but of course we can also use `alt=""` and/or `role="presentation"` and/or `aria-hidden="true"` etc. wherever necessary.

**Comment by Scarsniik** [comment]
Oh, ok
I'll do that

**Comment by Scarsniik** [comment]
Fixed in #630
